inequality/pen.py

```diff
@@ -0,0 +1,240 @@
+"""
+Pen's Parade and Pengram Visualizations
+
+This module provides functions to create Pen's Parade visualizations and
+extend them with choropleth maps to display the spatial distribution of
+values. The `pen` function generates a traditional Pen's Parade, which is
+a visual representation of income distribution or similar data, typically
+used to show inequality. The `pengram` function enhances this by combining
+the Pen's Parade with a choropleth map, allowing for a richer analysis of
+spatial data distributions.
+
+Author
+------
+Serge Rey <srey@sdsu.edu>
+"""
+
+import seaborn as sns
+import matplotlib.pyplot as plt
+import mapclassify as mc
+import matplotlib.patches as patches
+import math
+import pandas as pd
+import numpy as np
```

Comment:
We now depend only on lib, numpy and scipy here. All of these will either need to be imported as soft deps in the function or added as required to pyproject. 

The only thing I would be hesitant depending on is seaborn. Do we need it at all or can we use matplotlib/pandas to do the bar plot instead?